tokens/functional/shadow/light.json5

```diff
@@ -0,0 +1,57 @@
+{
+  shadow: {
+    small: {
+      $value: '{base.shadow.small}',
```

Comment:
Do we need to have shadows defined in `base` AND `functional`? It seems for now the ones in `functional` are just aliasing the ones in `base` and there isn't really a difference. `base.shadow.small` = `functional.shadow.small`.

If we wanted to change it to just a single place, I guess we should only have shadows under `functional` since that's what people are supposed to used in components.

Reply:
I am very open about this. We could define all shadows only as `functional` tokens. One could argue there are no base shadows. 

I am open for either way.

Reply:
> One could argue there are no base shadows.

Yeah, IMO we don't need `base` tokens if we only alias them in `functional`. We could add shadows to `base` if we had many shadows like we do with color scales. E.g. `base.shadow.[1-9]`. But yes, maybe we can only add them if we need it one day.